types/markdown-magic/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for markdown-magic 1.0
+// Project: https://github.com/DavidWells/markdown-magic#readme
+// Definitions by: Piotr Błażejewicz (Peter Blazejewicz) <https://github.com/peterblazejewicz>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.4
```

Comment:
This code doesn't look like it uses complicated types, so it probably doesn't require 3.4.

Reply:
Yup, read. Will do this asap, thanks!